robo4j-core/src/test/java/com/robo4j/units/StringProducerRemote.java

```diff
@@ -54,6 +57,6 @@ public void onMessage(T message) {
         int totalMessages = totalCounter.incrementAndGet();
         RoboContext ctx = LookupServiceProvider.getDefaultLookupService().getContext(targetContext);
         ctx.getReference(target).sendMessage(message);
-        System.out.println(String.format("class: %s, totalMessages: %d",getClass().getSimpleName(), totalMessages));
+        System.out.printf("class: %s, totalMessages: %d%n",getClass().getSimpleName(), totalMessages);
```

Comment:
I would prefer to replace all System.out with a logger...

Reply:
we will do so, robo4j has internal logger util, I've created a ticket we need to review it